### ISSUE TITLE
Tutorial for Hamming loss

### ISSUE THREAD

**Issue by harshithere** [tensorflow/addons#1979]
I recently used Hamming loss from tfa for my thesis project. If it sounds useful, I'd like to add a tutorial on how to use the metric and probably other metrics in general. It took me some time to get around it so maybe this could save some time for others. Do let me know if this sounds like a worthwhile addition

- Are you willing to contribute it: Yes
- Are you willing to maintain it going forward?: In the near future, yes
- Is there a relevant academic paper?: No
- Is there already an implementation in another framework?: Requesting a tutorial 
- Was it part of tf.contrib?: No

**Which API type would this fall under - Metrics

People who are working on multi-label classification problems would benefit from such a tutorial as tfa is the only library I could find for using these metrics while model training in tensorflow/keras.

P.S - I haven't done many open-source contributions and I'm still learning

**Comment by bhack** [comment]
You are welcome. Take an overview on https://github.com/tensorflow/addons/tree/master/docs/tutorials

**Comment by AakashKumarNain** [comment]
Hi @harshithere , sounds good to me. To start with, please check out the `contributing` guidelines [here](https://github.com/tensorflow/addons#contributing). We add tutorials in the form of `notebooks`. Check out these [guidelines](https://github.com/tensorflow/addons/blob/master/docs/tutorials/README.md) for the same.

**Comment by harshithere** [comment]
Perfect!!
I had a look at the tutorials earlier. I'll get started soon and let you know if I need any help. Thanks @bhack and @AakashKumarNain

**Comment by harshithere** [comment]
https://github.com/tensorflow/addons/pull/2044
@bhack , @AakashKumarNain  I submitted a pull request for the same. Looking forward to your reviews

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
